openslides_backend/services/datastore/adapter.py

```diff
@@ -201,7 +206,9 @@ def get_all(
                     raise DatastoreException(
                         "Response from datastore contains invalid 'meta_position'."
                     )
-                collection_field = CollectionField(collection, field)
+                collection_field = cast(
+                    CollectionField, f"{collection}{KEYSEPARATOR}{field}"
+                )
```

Comment:
extract into helper method, maybe there are more places who need this?

Reply:
done